mbf_abstract_nav/src/recovery_action.cpp

```diff
@@ -83,7 +83,7 @@ void RecoveryAction::run(GoalHandle &goal_handle, AbstractRecoveryExecution &exe
         if (execution.isPatienceExceeded())
         {
           ROS_INFO_STREAM("Recovery behavior \"" << goal.behavior << "\" patience exceeded! Cancel recovering...");
-          if (!execution.cancel())
+          execution.cancel();
           {
             ROS_WARN_STREAM("Cancel recovering \"" << goal.behavior << "\" failed or not supported; maybe wait until it is finished!");
           }
```

Comment:
These lines should also be removed, right?

Reply:
:see_no_evil: